packages/search/tests/integration/components/navi-search-result/asset-test.js

```diff
@@ -0,0 +1,44 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, findAll } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import Service from '@ember/service';
+import { set } from '@ember/object';
+
+module('Integration | Component | navi-search-result-asset', function(hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+
+  let service;
+
+  hooks.beforeEach(async function() {
+    // Load metadata needed for request fragment
+    await this.owner.lookup('service:bard-metadata').loadMetadata();
+    service = this.owner.lookup('service:navi-search-provider');
+    const store = this.owner.lookup('service:store'),
+      mockAuthor = store.createRecord('user', { id: 'ciela' });
+    this.owner.register(
+      'service:user',
+      Service.extend({
+        getUser: () => mockAuthor
+      })
+    );
+  });
+
+  test('displays results', async function(assert) {
```

Comment:
maybe there should be a `no data` rendering test for this as well

Reply:
when results are empty

Reply:
There's never going to be the case where this component will not have data. Since we only instantiate it if there are results returned.

Reply:
👍

Reply:
can we dis-allow the case where results are empty, we talked about having an assert